molecules/data/dataloaders/dataset.py

```diff
@@ -9,10 +9,13 @@ def __init__(self, path, name=None):
         self.name = name
 
     def __repr__(self):
-        if self.name:
-            identifier = f'Data handler for {self.name} contact maps'
+        if self.name and sys.version[0] == 3:
+            identifier = 'Data handler for {self.name} contact maps'
             return identifier
-        return f'Data handler for contact maps'
+        elif self.name: 
+            identifier = 'Data handler for {} contact maps'.format(self.name)
+            return identifier
+        return 'Data handler for contact maps'
 
```

Comment:
Haha! You don't want to drag people into Python >= 3.6?

Reply:
I found it out installing the package with Python 2.7. Then we probably need to remove the dependency to Python 2.7 of the package. But some users, such as Sindhu, might not be too happy about it. Haha